File: CodeForces/1601-1700/1676/A/1676A.py
# Problem 1676A: Lucky?
# https://codeforces.com/contest/1676/problem/A
# Executed with: Python 3.11.2
# Linux version: Linux kernel 6.1.0-10-amd64
# Submitted on: September 26th, 2024
# Codeforces language used: PyPy 3.10 (7.3.15, 64bit)

tickets = int(input())
for _ in range(0,tickets):
    ticket = [int(char) for char in str(input())]
    if sum(ticket[:3]) == sum(ticket[3:]):
        print("YES")
    else:
        print("NO")

File: CodeForces/1601-1700/1676/A/1676A.cpp
// Problem 1676A: Lucky?
// https://codeforces.com/contest/1676/problem/A
// Compiled with: g++ (Debian 12.2.0-14+deb12u1) 12.2.0
// Linux version: Linux kernel 6.1.0-10-amd64
// Submitted on: October 22nd, 2025
// Codeforces language used: C++23 (GCC 14-64, msys2)

#include <iostream>
#include <string>
using namespace std;

int main() {
    int t;
    cin >> t;

    for(int i = 0; i < t; i++) {
        string ticket;
        cin >> ticket;
        int total = 0;
        total += ticket[0];
        total += ticket[1];
        total += ticket[2];
        total -= ticket[3];
        total -= ticket[4];
        total -= ticket[5];

        if (total == 0) {
            cout << "YES\n";
        } else{
            cout << "NO\n";
        }
    }
}
